web/components/App.jsx

```diff
@@ -65,47 +87,32 @@ class App extends React.Component {
       const newValue = this.state.value;
       newValue[role.label] = values;
       this.setState({ value: newValue });
+
+      values = (values ? values : '');
```

Comment:
`values = values || '';` might be better, to be consistent with line 82, `let values = inputValue || null;`

Reply:
true, always forget about this js feature.

Reply:
honestly, I have no idea why this ternary exist